docs/usage/getting-started/running.md

```diff
@@ -0,0 +1,209 @@
+Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+Don't forget to configure `platform=gitlab` somewhere in config.
+
+#### Bitbucket Cloud
+
+First, [create an AppPassword](https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html) for the bot account.
+Configure it as `password` in your `config.js` file, or in environment variable `RENOVATE_PASSWORD`, or via CLI `--password=`.
+Also be sure to configure the `username` for your bot account too.
+Don't forget to configure `platform=bitbucket` somewhere in config.
+
+#### Bitbucket Server
+
+Create a [Personal Access Token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html) for your bot account.
+Configure it as `password` in your `config.js` file, or in environment variable `RENOVATE_PASSWORD`, or via CLI `--password=`.
+Also configure the `username` for your bot account too, if you decided not to name it `@renovate-bot`.
+Don't forget to configure `platform=bitbucket-server` somewhere in config.
+
+If you use MySQL or MariaDB you must set `unicodeEmoji` to `false` in the bot config (`RENOVATE_CONFIG_FILE`) to prevent issues with emojis.
+
+### Azure DevOps
+
+First, [create a Personal Access Token](https://docs.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/pats) for the bot account.
+Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+Don't forget to configure `platform=azure` somewhere in config.
+
+#### Gitea
+
+First, [create a access token](https://docs.gitea.io/en-us/api-usage/#authentication-via-the-api) for your bot account.
+Configure it as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+Don't forget to configure `platform=gitea` somewhere in config.
+
+### GitHub.com token for release notes
+
+If you are running on any platform except github.com, it's important to also configure the environment variable `GITHUB_COM_TOKEN` containing a Personal Access Token for github.com.
+This account can actually be _any_ account on GitHub, and needs only read-only access.
+It's used when fetching release notes for repositories in order to increase the hourly API limit.
+It's also OK to configure the same as a host rule instead, if you prefer that.
+
+**Note:** If you're using Renovate in a project where dependencies are loaded from github.com (such as Go modules hosted on GitHub) it is highly recommended to add a token as you will exceed the rate limit from the github.com API, which will lead to Renovate closing and reopening PRs because it could not get reliable info on updated dependencies.
+
+### Self-hosting examples
+
+For more examples on running Renovate self-hosted, please read our [Self-hosted examples](../examples/self-hosting.md) page.
```

Comment:
This link doesn't work from the GitHub rich-preview, will it work on the published site?

Reply:
I hope so, although ideally we'd have a link approach which works on both